RELEASE.md

```diff
@@ -125,15 +129,22 @@ Please make sure you have these tools installed before starting this process:
 
     Involve the Solutions Engineers stakeholders in the release process.
 
-    Remind them that this release candidate is available for testing in the [release environment](./README.md#Development).
+    Inform them that this release candidate is available for testing in the [release environment](./README.md#Development). At this point the Solutions Architects should:
 
-    At this point it may be relevant to test a fresh installation of the release candidate as well as an upgraded installation.
+    -   [ ] 11.1.  Perform a fresh installation of the release candidate
+    
+    -   [ ] 11.2.  Perform an upgrade on an existing installation
 
-    Should all of these stakeholders be happy, a go-ahead of the release should be given by clicking the Manual Approval step for the Solutions Engineers in the CircleCI workflow of your release branch.
+    -   [ ] 11.3.  Should all of these stakeholders be happy, a go-ahead of the release should be given by clicking the Manual Approval   step for the Solutions Engineers in the CircleCI workflow of your release branch.
     
     After this Manual Approval on CircleCI the workflow will promote the RC to the [stable](https://charts.codacy.com/stable/api/charts) channel.
 
-    If all is good give a public OK to the release.
+-   [ ] 12.  If all is good give a public OK to the release
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-indent] Incorrect list-item indent: remove 1 space](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=5619697)

Reply:
We should improve the `.remarkrc` file to allow indenting list items with 4 spaces. 

On the codacy/chart repository we have this really simple file that includes the [remark-preset-lint-recommended](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-preset-lint-recommended) rule:

https://github.com/codacy/chart/blob/master/.remarkrc

However, currently the `.remarkrc` file [for the Handbook](https://github.com/codacy/handbook/blob/master/.remarkrc.js) is much more complex. :sweat_smile: